carma_wm/src/CARMAWorldModel.cpp

```diff
@@ -263,11 +263,23 @@ std::vector<lanelet::ConstLanelet> CARMAWorldModel::getLaneletsBetween(double st
     TrackPos min = routeTrackPos(front);
     TrackPos max = routeTrackPos(back);
 
-    if (std::max(min.downtrack, start) > std::min(max.downtrack, end))
-    {  // Check for 1d intersection
-      // No intersection so continue
-      continue;
+    if (!bounds_inclusive) // reduce bounds slightly to avoid including exact bounds
+    {
+      if (std::max(min.downtrack, start + 0.1) > std::min(max.downtrack, end - 0.1))
```

Comment:
Let's use smaller bounds like 0.00001 for the comparison as 0.1 is a common increment size developers use. 
Additionally, the bound size needs to be documented in this methods comments to let users know that the non-inclusive behavior is not equivalent to a != check.

Reply:
Reduced the number and added the comment in headerfile